recipes/libmysqlclient/all/conanfile.py

```diff
@@ -24,67 +24,43 @@ class LibMysqlClientCConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
-        "with_ssl": [True, False, "deprecated"],
-        "with_zlib": [True, False, "deprecated"],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
-        "with_ssl": "deprecated",
-        "with_zlib": "deprecated",
     }
-
+    generators = "VirtualRunEnv", "VirtualBuildEnv", "CMakeDeps"
```

Comment:
Not like this, you want fine-grained control of order and scope where these scripts will be called